readthedocs/search/documents.py

```diff
@@ -159,7 +160,7 @@ def get_queryset(self):
         # Also do not index certain files
         queryset = queryset.filter(
             project__documentation_type__contains='sphinx'
-        )
+        ).exclude(version__type=PULL_REQUEST)
```

Comment:
can this not use the manager? If not, we should probably add a queryset method that does the same thing, so we can use it in cases like this.

Reply:
@ericholscher Added Queryset for this